internal/controller/apiextensions/composite/composition_ptf_test.go

```diff
@@ -52,9 +52,9 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/resource/unstructured/composite"
 	"github.com/crossplane/crossplane-runtime/pkg/test"
 
-	"github.com/crossplane/crossplane/apis/apiextensions/fn/io/v1alpha1"
 	iov1alpha1 "github.com/crossplane/crossplane/apis/apiextensions/fn/io/v1alpha1"
 	fnpbv1alpha1 "github.com/crossplane/crossplane/apis/apiextensions/fn/proto/v1alpha1"
+	fnv1alpha1 "github.com/crossplane/crossplane/apis/apiextensions/fn/proto/v1alpha1"
```

Comment:
do we need to import the same package twice, under two different aliases?

Reply:
No, this is an oversight. Will fix.

Reply:
is it possible to catch up this with some linting rule in the future?